docs/site-replication/run-sse-kms-object-replication.sh

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC2120
+exit_1() {
+	cleanup
+
+	echo "minio1 ============"
+	cat /tmp/minio1_1.log
+	echo "minio2 ============"
+	cat /tmp/minio2_1.log
+
+	exit 1
+}
+
+cleanup() {
+	echo -n "Cleaning up instances of MinIO ..."
+	pkill -9 minio || sudo pkill -9 minio
+	pkill -9 kes || sudo pkill -9 kes
+	rm -rf ${PWD}/keys
+	rm -rf /tmp/minio{1,2}
+	echo "done"
+}
+
+cleanup
+
+export MINIO_CI_CD=1
+export MINIO_BROWSER=off
+export MINIO_ROOT_USER="minio"
+export MINIO_ROOT_PASSWORD="minio123"
+
+# Create certificates for TLS enabled MinIO
+echo -n "Setup certs for MinIO instances ..."
+wget -O certgen https://github.com/minio/certgen/releases/latest/download/certgen-linux-amd64 && chmod +x certgen
+./certgen --host localhost
+mkdir -p ~/.minio/certs
+mv public.crt ~/.minio/certs || sudo mv public.crt ~/.minio/certs
+mv private.key ~/.minio/certs || sudo mv private.key ~/.minio/certs
+echo "done"
+
+# Deploy KES and start
+echo "Setup KES for MinIO instances ..."
+wget -O kes https://github.com/minio/kes/releases/download/2024-03-13T17-52-13Z/kes-linux-amd64 && chmod +x kes
+./kes identity new --key private.key --cert public.crt --ip "127.0.0.1" localhost --force
+./kes identity new --key=minio.key --cert=minio.crt MinIO --force
+MINIO_KES_IDENTITY=$(kes identity of minio.crt | awk NF)
+cat >kes-config.yml <<EOF
+address: 0.0.0.0:7373
+
+admin:
+  identity: disabled
+
+tls:
+  key: private.key
+  cert: public.crt
+
+policy:
+  my-app:
+    allow:
+    - /v1/key/list/*
+    - /v1/key/generate/*
+    - /v1/key/decrypt/*
+    - /v1/key/create/*
+    - /v1/secret/list/*
+    - /v1/status
+    identities:
+    - ${MINIO_KES_IDENTITY}
+
+keystore:
+  fs:
+    path: ./keys
+EOF
```

Comment:
you don't need kes, although I appreciate the effort here. You can simply use MINIO_KMS_SECRET_KEY instead

Reply:
oh yes